tests/entities/test_param.py

```diff
@@ -1,28 +1,35 @@
-import unittest
-
+import pytest
 from mlflow.entities import Param
 from tests.helper_functions import random_str, random_int
 
 
-class TestParam(unittest.TestCase):
-    def _check(self, param, key, value):
-        assert isinstance(param, Param)
-        assert param.key == key
-        assert param.value == value
+def _check(param, key, value):
+    assert isinstance(param, Param)
+    assert param.key == key
+    assert param.value == value
+
+
+@pytest.fixture(scope="module")
+def key():
+    yield random_str(random_int(10, 25))  # random string on size in range [10, 25]
```

Comment:
I think we can directly use `random_str(random_int(10, 25))`.

Reply:
solved in https://github.com/wkrt7/mlflow/commit/3a76eb2e91490a13ea70ca338c5a84dd491f8c26

Reply:
@wkrt7 Can we also remove `key` and `value`?

Reply:
I moved key and value to the test function